domain/src/test/groovy/com/github/maleksandrowicz93/educational/institution/EducationalInstitutionSpec.groovy

```diff
@@ -0,0 +1,133 @@
+package com.github.maleksandrowicz93.educational.institution
+
+import spock.lang.Specification
+
+class EducationalInstitutionSpec extends Specification {
+
+    def "faculty may be created"() {
+        given: "Educational Institution without any faculty"
+        and: "new faculty to be created"
```

Comment:
`new faculty to be created` brzmi jak przygotowanie obiektu, który ma zostać utworzony przed przekazaniem go do metody (czyli bardziej jak sekcja `given`) - zamiast tego proponuję `new faculty is created` albo `new faculty creation is triggered` jako opis akcji w `when`

teraz widze, że  tutaj jest tylko `given` i `expect` - brakuje bloku `when`

Reply:
Właśnie o to mi chodziło -  w given przygotowujemy obiekt, który będziemy chcieli przekazać do metody agregatu.

When nie brakuje, expect działa jak when + then jednocześnie, przydatne do weryfikacji operacji w stylu
```groovy
expect:
Math.max(1, 2) = 2 
```
lub właśnie
```groovy
expect:
agregat.utworzCos()
```

Reply:
a gdzie sprawdzasz, że jest utworzone i że utworzyło się poprawnie?